docsite/index.html

```diff
@@ -1497,149 +1497,9 @@ <h5>Prefers Dark</h5>
                   :root {
                     --brand: var(--brand-dark);
 
-                    --text1: var(--text1-dark);
-                    --text2: var(--text2-dark);
+                    --text-1: var(--text-1-dark);
+                    --text-2: var(--text-2-dark);
 
-                    --surface1: var(--surface1-dark);
-                    --surface2: var(--surface2-dark);
-                    --surface3: var(--surface3-dark);
-                    --surface4: var(--surface4-dark);
-                  }
-                }
-              </code></pre>
-            </div>
-            <div>
-              <h5>Prefers Dim</h5>
-              <pre class="language-css"><code>
-                @media (prefers-contrast: less) {
-                  :root {
-                    --brand: var(--brand-dim);
-
-                    --text1: var(--text1-dim);
```

Comment:
how come this whole chunk is deleted?

Reply:
Sorry, I edited online an maybe some copy and paste was the cause. I will fix it